Apps/gladys/docker-compose.yml

```diff
@@ -11,50 +11,19 @@ services:
     image: gladysassistant/gladys:v4 # Docker image to use for the 'app' service
     restart: always # Container should always restart
     privileged: true # Grants additional privileges to this container
+    network_mode: host
+    container_name: gladys
+    cgroup: host
     environment: # Environment variables for the container
       NODE_ENV: production # Sets the environment to production
       SQLITE_FILE_PATH: /var/lib/gladysassistant/gladys-production.db # Path to the SQLite database
+      TZ: Europe/Paris # Timezone
+      SERVER_PORT: "1080" # Server port for UI
     volumes: # Mount points from the host to the container
       - /var/run/docker.sock:/var/run/docker.sock # Docker socket file for Docker within Docker
-      - /DATA/AppData/$AppID/data:/var/lib/gladysassistant # Persistent storage for Gladys data
+      - /var/lib/gladysassistant:/var/lib/gladysassistant # Persistent storage for Gladys data
       - /dev:/dev # Access to host devices
-    ports: # Port mappings from the host to the container
-      - "1080:80" # Maps port 80 in the container to port 1080 on the host
-      - "10443:443" # Maps port 443 in the container to port 10443 on the host
-
-    x-casaos: # CasaOS specific configuration
```

Comment:
The x-casaos needs to stay with describing the envs and volumes.

Reply:
Fixed